converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/TableStorage/TableErrorType.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.DicomCast.Core.Features.ExceptionStorage
+{
+    /// <summary>
+    /// Represents the type of error that is going to be stored
+    /// </summary>
+    public enum TableErrorType
```

Comment:
This type doesn't seem to have anything to do with the tables, and is more of an error type or classification.

Reply:
Changed the name